whois-api/src/main/java/net/ripe/db/whois/api/rdap/RdapObjectMapper.java

```diff
@@ -161,7 +160,11 @@ public RdapObject mapHelp(final String requestUrl) {
         return mapCommons(new RdapObject(), requestUrl);
     }
 
-    private RdapObject getRdapObject(final String requestUrl, final RpslObject rpslObject, final LocalDateTime lastChangedTimestamp, @Nullable final RpslObject abuseContact) {
+    private RdapObject getRdapObject(final String requestUrl,
+                                     final RpslObject rpslObject,
+                                     final LocalDateTime lastChangedTimestamp,
+                                     @Nullable final RpslObject abuseRoleObject,
```

Comment:
Mixing Nullable and Optional in the same method arguments? Can we choose one style for consistency?

Reply:
We use just the AbuseContact now.